Algorithms/JavaScript/fibonacci-number.js

```diff
@@ -1,7 +1,17 @@
+// Return the nth fibonacci number.
 function fib(n){
```

Comment:
I'd rename to `fibonacci`. It's also much more efficient to compute iteratively than recursively.